core/src/main/java/org/apache/calcite/prepare/CalcitePrepareImpl.java

```diff
@@ -943,7 +943,7 @@ private int getTypeOrdinal(RelDataType type) {
   }
 
   private static String getClassName(RelDataType type) {
-    return null;
+    return Object.class.getName();
```

Comment:
Can we add a comment to explain this behavior?

Reply:
Will do